app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/snippet/CreateSnippetConverter.kt

```diff
@@ -0,0 +1,20 @@
+package com.bottlerocketstudios.brarchitecture.ui.snippet
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import com.bottlerocketstudios.compose.snippets.CreateSnippetScreenState
+
+@Composable
+fun CreateSnippetFragmentViewModel.toState() = CreateSnippetScreenState(
+    title = title.collectAsState(),
+    filename = filename.collectAsState(),
+    contents = contents.collectAsState(),
+    private = private.collectAsState(),
+    failed = failed.collectAsState(),
```

Comment:
@VanyaHuamanBottleRocket @BottleRocket-Colin For clarity's sake, how do you feel about `creationFailed` to give a clue to the cause of the failure (although the class name does help some with this already)?

Reply:
Renaming